### ISSUE TITLE
Railway Deployment Issues

### ISSUE THREAD

**Issue by idont85** [titaniumnetwork-dev/Ultraviolet-Node-Archive#30]
When attempting to deploy to the Railway platform, the deployment succeeds but soon after crashes. I cannot figure out how to fix it.
## Expected Behavior

It should highlight the link in green and should let it work like usual.
## Current Behavior

Instead there is a server error and it doesn't work.
## Possible Solution

I am not too good at web coding, but I think it may not be configured for Railway.
## Steps to Reproduce

1. Fork this github
2. Go to Railway and begin deployment
3. Wait...
4. Watch your failure take place when you click the link.

## Context (Environment)

Trying to deploy.

**Comment by noonoo5084** [comment]
you have to go to the readme.md and remove anything that says youtube for osme reason it blocks any project with youtube in it

**Comment by idont85** [comment]
Well, it deploys, but crashes within seconds...

**Comment by noonoo5084** [comment]
yea you have to use a start command thing in the setting which i havent fully learned how to use lmk if u figure it out

**Comment by idont85** [comment]
Yeah, thanks for responding. Tried tinkering with that earlier but to no success...
